team.html

```diff
@@ -1,333 +1,472 @@
+        </p>
+        <p>
+            We expect this structure to evolve over the coming years, adapting to
+            the size and composition of the Gammapy development team, and the requirements and needs of scientists and
+            projects using Gammapy. If you would like to become part of the Gammapy team, please get in <a
+                href="contact.html">contact</a>.
+            Help is always welcome!
+        </p>
+        <h2>Overview</h2>
+        <p>
+            The following sections describe the major roles and responsibilities in the Gammapy team:
+        </p>
+        <ul>
+            <li><a href="#coordination-committee">Coordination committee</a></li>
+            <li><a href="#project-manager">Project manager</a></li>
+            <li><a href="#lead-developer">Lead developer</a></li>
+            <li><a href="#core-developers">Core developers</a></li>
+            <li><a href="#contributors">Contributors</a></li>
+            <li><a href="#other">Other</a></li>
+        </ul>
+
+        <h2 id="cc">Coordination Committee</h2>
+        <p class="lead">
+            The Gammapy coordination committee (CC) is the board that is responsible to promote, coordinate and steer
+            Gammapy developments. It also serves as main contact point for the Gammapy project.
+        </p>
+        <p>
+            Gammapy is developed and used by people and projects from several institutes and countries, with different
+            needs, priorities and schedules. Members of the CC are representatives for the major stakeholders, i.e.
+            groups that have contributed significantly to Gammapy development, as well as projects like H.E.S.S. and CTA
+            that are using Gammapy. In addition, the Gammapy project manager and lead developer are part of the Gammapy
+            CC.
+        </p>
+        <p>
+            Responsibilities include:
+        </p>
+        <ul>
+            <li>
+                Ensure that scientists and projects that are using Gammapy have reliable, good and long-term support
+            </li>
+            <li>
+                Promote the use of Gammapy by new projects, especially as science tool for CTA
+            </li>
+            <li>
+                Support and grow the Gammapy team (help find manpower and funding)
+            </li>
+            <li>
+                Make decisions on the scope, content and development priorities for the Gammapy package. <br>
+                In many cases this will be done via documents we call "proposals for improvement of Gammapy (PIGs)".
+            </li>
+            <li>
+                Support and coordinate the use of Gammapy for scientific or technical studies and papers
+            </li>
+            <li>
+                The CC is the official contact point for the Gammapy project.
+            </li>
+            <li>
+                Appoint the Gammapy project manager (technical lead) and lead developer (non-technical lead)
```

Comment:
the project manager is the non-technical lead and the lead developer is the technical lead, as stated in the paragraphs below